derive_client/_bridge/w3.py

```diff
@@ -238,7 +246,11 @@ async def estimate_fees(w3, blocks: int = 20) -> FeeEstimates:
     estimates = {}
     for percentile in percentiles:
         rewards = percentile_rewards[percentile]
-        estimated_priority_fee = int(statistics.median(rewards))
+        non_zero_rewards = list(filter(lambda x: x, rewards))
+        if non_zero_rewards:
+            estimated_priority_fee = int(statistics.median(non_zero_rewards))
+        else:
+            estimated_priority_fee = MIN_PRIORITY_FEE
```

Comment:
we filter out zero values, as we can end up estimating the median to be zero. In case there are no `non_zero_rewards`, we set a `MIN_PRIORITY_FEE`, currently set at `10_000`